apidoc/Titanium/XML/ProcessingInstruction.yml

```diff
@@ -1,23 +1,22 @@
 ---
 name: Titanium.XML.ProcessingInstruction
 extends: Titanium.Proxy
-summary: A way to keep processor-specific information in the text of the document. Implements the 
-    [DOM Level 2 API](http://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-1004215813) 
-    on Android and iOS. Exposes the 
-    [DOM Level 3 API](http://www.w3.org/TR/DOM-Level-3-Core/core.html#ID-1004215813) implementation on Mobile Web.
+summary: A way to keep processor-specific information in the text of the document. Implements the
+    [DOM Level 2 API](http://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-1004215813)
+    on Android and iOS.
 since: "0.9"
 createable: false
-platforms: [android, iphone, ipad, mobileweb]
+platforms: [android, iphone, ipad, ]
```

Comment:
I believe the YML is Ok with trailing commas in arrays like this, but I can fix them up.